src/code/z_camera.c

```diff
@@ -7277,6 +7281,7 @@ void Camera_PrintSettings(Camera* camera) {
         sp48[i] = '\0';
 
         sp48[camera->play->activeCamId] = 'a';
+#if OOT_DEBUG
         DebugCamera_ScreenTextColored(3, 22, DEBUG_CAM_TEXT_WHITE, sp58);
```

Comment:
surprisingly, `DebugCamera_ScreenTextColored` is in `z_debug.c`, and double surprisingly `z_debug.o` is linked into the retail rom, so you don't need to ifdef it out

Reply:
Huh. Removed these ifdefs